README.md

```diff
@@ -12,18 +12,18 @@ To build just the vocoder itself you will need:
     - Debian
     - ALSA
 - Hardware
-    - Raspberry Pi 3 (or newer) running a 64bit OS
+    - Raspberry Pi 3 (or newer) running a 64bit OS (we used Rasbian). For improved performance a Raspberry Pi 4 is recommended.
```

Comment:
This is too long for a bullet point, we are also currently running it fine on a pi3. Would remove

Reply:
Moved the Pi 4 comment